torchvision/transforms/functional_tensor.py

```diff
@@ -997,8 +1014,8 @@ def rotate(
             If true, expands the output image to make it large enough to hold the entire rotated image.
             If false or omitted, make the output image the same size as the input image.
             Note that the expand flag assumes rotation around the center and no translation.
-        fill (n-tuple or int or float): this option is not supported for Tensor input.
-            Fill value for the area outside the transform in the output image is always 0.
+        fill (sequence or int or float, optional): Optional fill value, default None.
```

Comment:
Docstring updates can be ignored for private modules like torchvision/transforms/functional_tensor.py and torchvision/transforms/functional_pil.py

Reply:
So I can leave those as they are now or roll them back to before these commits?

Reply:
Please, keep them as they are now. Your modification is OK.